.travis.yml

```diff
@@ -10,6 +10,9 @@ rust:
   - nightly
   - beta
   - stable
+env:
+  - CARGO_EXTRA_ARGS="--all-features"
+  - CARGO_EXTRA_ARGS=""
```

Comment:
Is that a mistake or is it duplicated intentionally?

Reply:
Intentional so it does both. So matrix of (nightly, beta, stable) * (standard features, all features).